### ISSUE TITLE
slosh needs ability to run *some* tests in Rust

### ISSUE THREAD

**Issue by gpwclark** [sl-sh-dev/sl-sh#114]
In a lisp ideally as much of the language utilities as possible are written in the host language.

However, this concept is in tension with the idea that you shouldn't be able to have a project compile, and run tests if it's in a broken state.

Therefore. I would like a way to test the builtins written in rust when `cargo test` is run. This doesn't mean functions written in slosh with examples in slosh should be tested by cargo test. But it does mean that if `cargo test` passed and slosh tests fail, there isn't something "fundamentally" broken about the language that's causing slosh tests to fail. If w/e slosh test runner is written fails the idea is there is as much of a guarantee as possible that the problem is in lisp code, not in rust code.

tldr;
I think this translates to.... make an integration test that calls the slosh test script.

**Comment by gpwclark** [comment]
resolved in PR #111
